travis/install_avr32-gcc.sh

```diff
@@ -0,0 +1,20 @@
+mkdir avr32-gcc
+cd avr32-gcc
+
+# Get archives
+wget https://gist.github.com/jlecoeur/d83107dda77c4bd62a66/raw/1dc9c3efe832db7cfcd86694693f620d811e9802/avr32-gnu-toolchain-3.4.2.435-linux.any.x86.tar.gz
+wget https://gist.github.com/jlecoeur/d83107dda77c4bd62a66/raw/471a24180afc9c799848e017aa2f8cf10c55cb1a/atmel-headers-6.1.3.1475.zip
```

Comment:
It is not possible to create a gist account for lis-epfl? 
If not, I agree that the 3rd option is for now the most suited one.